salt/states/pkg.py

```diff
@@ -1241,10 +1245,14 @@ def installed(
     failed_hold = None
     if targets or to_reinstall:
         reinstall = bool(to_reinstall)
+        force = False
+        if salt.utils.is_freebsd():
+            force = True    # Downgrades need to be forced.
```

Comment:
OK, this wasn't mentioned at all in the pull request message, so I wasn't sure. Thanks for the clarification.